### PR TITLE
Multiple table

### PR DESCRIPTION
**Pull request by johnie261**
Hi there,
### I did the following:

- Created owners table and species table.
- Modified animals table.
- Added data in owners and species table.
- Modified insert animals.
- Below are screenshots of queries answering the following questions 
>  What animals belong to Melody Pond?
Screenshot from 2023-05-03 23-43-29
> List of all animals that are Pokemon (their type is Pokemon).
Screenshot from 2023-05-03 23-47-58
> List all owners and their animals, remember to include those that don't own any animal.
Screenshot from 2023-05-03 23-52-32
> How many animals are there per species?
Screenshot from 2023-05-03 23-55-28
> List all Digimon owned by Jennifer Orwell.
Screenshot from 2023-05-04 00-03-30
> List all animals owned by Dean Winchester that haven't tried to escape. (All his animals tried to escape)
Screenshot from 2023-05-04 00-06-53
> Who owns the most animals?
Screenshot from 2023-05-04 00-08-49

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -61,6 +61,42 @@ SELECT species, AVG(escape_attempts) from animals
...
+
+-- List all animals owned by Dean Winchester that haven't tried to escape
+SELECT a.name animal_name, a.escape_attempts, o.full_name as owner 
+FROM animals a 
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Dean Winchester' 
+AND a.escape_attempts = 0;
+
+
+
```
Comment (m05e5) [inline queries.sql]: - [ ] Great job adding your queries but it seems you have missed ( forgotten ) a query. please kindly make sure to add it
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @johnie261 , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (johnie261) [reply queries.sql]: Thank you, I forgot to add the query. let me add the query

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @johnie261, 

To highlight 👍🏽 
✔️ Queries are added
✔️ Screenshots are added

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**